js/create-highlight.sh

```
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+git clone https://github.com/highlightjs/highlight.js.git -b 11.11.0 --depth 1 highlight
+pushd highlight
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)